packages/@azure/servicebus/data-plane/test/renewLockSessions.spec.ts

```diff
@@ -27,7 +27,7 @@ describe("Standard", function(): void {
   const namespace = Namespace.createFromConnectionString(SERVICEBUS_CONNECTION_STRING);
 
   const STANDARD_QUEUE_SESSION =
-    process.env.QUEUE_NAME_NO_PARTITION_SESSION || "unpartitioned-queue";
+    process.env.QUEUE_NAME_NO_PARTITION_SESSION || "unpartitioned-queue-sessions";
```

Comment:
Why aren't we using the testUtils?

Reply:
Because this test file followed the model of renewLock.spec.ts which you had written in a different way i.e not using the testUtils to create the sender/receiver clients